tfx/experimental/pipeline_testing/base_stub_executor.py

```diff
@@ -0,0 +1,75 @@
+# Lint as: python2, python3
```

Comment:
Please drop python2 here.
Because `FileNotFoundError` which doesn't exists in Python2 used in this file, it won't work with Python2. Anyway we don't need to support python2 any more in new codes.